src/components/Panel/Channels/index.vue

```diff
@@ -140,6 +144,16 @@ export default {
     },
   },
 
+  created () {
+    this.$MessagingAPI.permissionsEffective().then(e => {
```

Comment:
Move API comm out of components into views and use vuex.
In messaging, we will move a couple of other things there as well so let's start preparing a good environment for that.

Lets's build a vuex store (`session` looks like a good name for it) that will keep the stuff we need for current session. Permissions are one of the the first thing and adding an object of actions (current) user is allowed to do (`{ channelGroupCreate: bool }`). Add it under `permissions`.
Add loading of it under `init` action and add all mutations we need. Base upon other, existing stores (`pending` state).

Then add (and use) simple getters (`canCreateGroupChannel()`...).

Reply:
Deal. This is the next step.